onsite/lucxor/cli.py

```
@@ -935,71 +988,119 @@ def run(
+            best_sequence = psm.get_best_sequence(include_decoys=False)
+            seq_str = best_sequence or psm.peptide.peptide
+            base_seq = psm.peptide.peptide  # fallback unmodified
+
+            # Use pyOpenMS to get unmodified string if possible
+            try:
+                from pyopenms import AASequence
+                seq_obj = AASequence.fromString(seq_str) if seq_str else None
+                if seq_obj:
+                    base_seq = seq_obj.toUnmodifiedString()
+            except Exception:
+                pass
```

Ruff (0.15.15)

[error] `try`-`except`-`pass` detected, consider logging the exception

(S110)